presto-main/src/test/java/com/facebook/presto/sql/planner/optimizations/TestReorderJoins.java

```diff
@@ -111,7 +111,7 @@ public void testEliminateCrossJoinPreserveFilters()
                         anyTree(
                                 join(INNER, ImmutableList.of(equiJoinClause("P_PARTKEY", "L_PARTKEY")),
                                         anyTree(PART_TABLESCAN),
-                                        anyTree(filter("L_RETURNFLAG = 'R'", LINEITEM_WITH_RETURNFLAG_TABLESCAN)))),
+                                        anyTree(filter("'R' = L_RETURNFLAG", LINEITEM_WITH_RETURNFLAG_TABLESCAN)))),
```

Comment:
yeah.  The PushDownTableConstraintsRewrite seems to switch the arguments to that predicate so the test failed